src/main/java/com/google/devtools/build/lib/runtime/commands/CqueryCommand.java

```diff
@@ -48,7 +48,7 @@
 @Command(
   name = "cquery",
   builds = true,
-  inherits = {BuildCommand.class},
+  inherits = {TestCommand.class},
```

Comment:
Quick comment rationale for why this value?

Reply:
Done, and added a test which failed before this PR - PTAL!